sechub-doc/src/test/java/com/daimler/sechub/restdoc/ProjectUpdateAdministrationRestControllerRestDocTest.java

```diff
@@ -95,6 +100,34 @@ public void restdoc_update_white_list_for_project() throws Exception {
 
 		/* @formatter:on */
 	}
+	
+	@Test
+	@UseCaseRestDoc(useCase=UseCaseUpdateProjectMetaData.class)
+	public void restdoc_update_metadata_for_project() throws Exception {
+
+		/* execute + test @formatter:off */
+        this.mockMvc.perform(
+        		post(https(PORT_USED).buildUpdateProjectMetaData(PROJECT_ID.pathElement()),"projectId1").
+        		contentType(MediaType.APPLICATION_JSON_VALUE).
+        		content("{\"apiVersion\":\"1.0\", \"metaData\":{\"key1\":\"value1\"}}")
+        		)./*andDo(print()).*/
+        			andExpect(status().isOk()).
+        			andDo(document(RestDocPathFactory.createPath(UseCaseUpdateProjectMetaData.class),
+        					pathParameters(
```

Comment:
Would it be possible to reduce the spacing a bit.

Reply:
done